R/ui.R

```diff
@@ -82,4 +82,8 @@ usei18n <- function(translator) {
 update_lang <- function(session, language) {
```

Comment:
Out of scope for this PR, but I wish `session` was the last argument with a default value of `shiny::getDefaultReactiveDomain()`. This way, the syntax is more consistent with Translator methods, e.g. `translate()`. Also, it will allow to simply pass the new language as a parameter most of the times :)

Reply:
Makes sense, issue added: https://github.com/Appsilon/shiny.i18n/issues/88